packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -866,7 +885,7 @@ class Auth {
      * A wrapper method for commerce-sdk-isomorphic helper: loginRegisteredUserB2C.
      *
      */
-    async loginRegisteredUserB2C(credentials: LoginRegisteredUserB2CCredentials) {
+    async loginRegisteredUserB2C(credentials: LoginRegisteredUserCredentialsWithCustomParams) {
```

Comment:
I believe this is non-breaking since `options` is optional

Reply:
Yes, that's correct. We had to be careful with this so that the change is non-breaking. We do that by adding an optional `options` into the `credentials` parameter. We did this rather than adding `options` as a separate parameter because the hook `login.mutateAsync()` will pass through only a single body parameter into `loginRegisteredUserB2C`.